leafmap/toolbar.py

```diff
@@ -205,7 +211,7 @@ def button_clicked(change):
         return toolbar_widget
 
 
-def tool_header_template(m=None, opened=True):
+def tool_header_template(m: Optional[geeMap] = None, opened: Optional[bool] = True):
```

Comment:
geemap in the soure code is a typo. They should be changed to leafmap

Reply:
but we still need to import leafmap right ?